assets/css/app.scss

```diff
@@ -28,3 +28,10 @@
   border: 1px solid #d8cccc;
   border-radius: 5px;
 }
+
+.navbar-brand {
+  font-family: 'Oswald', sans-serif;
+}
+.navbar-custom-bg {
+  background-color: #663399;
```

Comment:
Could we declare this as a variable either at the top of `app.scss` or in a new `colors.scss` which is then imported?